### PR TITLE
[API-805] App deauthorization webhooks docs

### PR DESCRIPTION
**Pull request by zackisaacs**
App deauth webhooks are coming. I'm writing these docs before finishing up the service implementation so that we can nail down the behavior and format of webhook events that get sent to external clients.

The only documentation change is that now the `object_type` can be either `athlete` or `activity`, and apps should be able to accept an `authorized: true` key-value pair for deauth events.

My plan is to merge this PR when the service implementation is complete.

Completes [API-805](https://strava.atlassian.net/browse/API-805)

### CODE REVIEW

File: content/docs/webhooks.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ To request access, email developers@strava.com with your client id, which you ca
 
 ## Event Data
 
-The Strava Webhook Events API supports webhook events for Strava activities.
+The Strava Webhook Events API supports webhook events for Strava activities, as well as athlete app deauthorizations.
 
 When an event occurs that corresponds to a push subscription, a POST request will be made to the callback url defined in the subscription.
-The payload contains the `object_type` and `aspect_type` for the updated object as well as the `object_id`, which is the activity's ID.
+The payload contains the `object_type` and `aspect_type` for the updated object, as well as an `object_id`, which is either an activity or athlete ID.
```
Comment (hariniriyer) [inline content/docs/webhooks.md]: super nit: The payload contains the `object_type` and `aspect_type` for the updated object, as well as an `object_id`, which is either an activity or an athlete ID.

File: content/docs/webhooks.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ These are the fields that are returned with webhook events:
       </span>
     </td>
     <td>
-        Always "activity."
+        Always either "activity" or "athlete."
```
Comment (hariniriyer) [inline content/docs/webhooks.md]: super duper nit: Always either "activity" or "athlete".

File: content/docs/webhooks.md (hunk 1 above)
Comment (zackisaacs) [reply content/docs/webhooks.md]: 👍

File: content/docs/webhooks.md (hunk 2 above)
Comment (zackisaacs) [reply content/docs/webhooks.md]: Going to have to push back on the `.` nit! I believe periods are supposed to go inside quotes to end a sentence: http://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html

**Comment by zackisaacs** [comment]
@MateoAO I was torn on whether to include `athlete`. I included it because I wanted it to be obvious that the athlete was the object associated with the event, but I agree it sounds unnatural.

**Comment by zackisaacs** [comment]
Squashed. Will merge once the implementation is finished.

**Pull request merged**